packages/grafana-ui/src/components/TimePicker/TimePickerContent/TimeRangeOption.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { memo } from 'react';
+import { css } from 'emotion';
+import { GrafanaTheme, TimeOption } from '@grafana/data';
+import { useTheme, stylesFactory, selectThemeVariant } from '../../../themes';
+
+const getStyles = stylesFactory((theme: GrafanaTheme) => {
+  const background = selectThemeVariant(
+    {
+      light: theme.colors.gray7,
+      dark: theme.colors.dark3,
+    },
+    theme.type
+  );
+
+  return {
+    container: css`
+      display: flex;
+      align-items: center;
+      justify-content: space-between;
+      padding: 7px 9px 7px 9px;
```

Comment:
These padding values are weird. I have pretty much the same component implemented in the Select PR (https://github.com/grafana/grafana/pull/21092/files#diff-6ebea9bb1f5cde1b0253d749fd4d54a4R41), and in the selects the list item has 8px padding. Maybe we could keep that consistent? And then rename this component to something more generic and use it in other places too (i.e. Select dropdown)?